src/Processors/QueryPlan/SortingStep.cpp

```diff
@@ -57,14 +60,14 @@ SortingStep::SortingStep(
 
 SortingStep::SortingStep(
     const DataStream & input_stream_,
-    SortDescription prefix_description_,
-    SortDescription result_description_,
+    const SortDescription & prefix_description_,
+    const SortDescription & result_description_,
```

Comment:
Why this change is needed? Passing by value + std::move will avoid potential copy.

Reply:
For example, here, if we add std::move to `order_descr`
https://github.com/ClickHouse/ClickHouse/blob/fe6533f41db18e7bb75a29f660c0fb2f68ee98d7/src/Interpreters/InterpreterSelectQuery.cpp#L1438

Reply:
Agree. It was oversimplification